### DISCUSSION TITLE
Works fine in local sever but when deployed throws  Err: Length octect is too long at shallow

### DISCUSSION THREAD

**Discussion by bytesagar** [nodejs/help#5039] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/43983588/216918444-cea62924-1c39-47ea-9b4a-eaf929e620bb.png)

**Comment by cjihrig** [comment, marked as answer]
If this works fine in a local server but breaks when deployed, this is likely not an issue with Node.js.
